superset-frontend/cypress-base/cypress/integration/dashboard/drilltodetail.test.ts

```diff
@@ -172,21 +172,21 @@ describe('Drill to detail modal', () => {
         openModalFromMenu('big_number_total');
         // checking the data
         cy.getBySel('row-count-label').should('contain', '75.7k rows');
-        cy.get('.virtual-table-cell').then($rows => {
+        cy.get('.virtual-table-cell').should($rows => {
           expect($rows).to.contain('Amy');
         });
         // checking the paginated data
         cy.get('.ant-pagination-item')
           .should('have.length', 6)
-          .then($pages => {
+          .should($pages => {
```

Comment:
Cool, I didn't know that `should` waits and `then` doesn't!

Reply:
Me either, this PR was a learning experience!